docs/index.md

```diff
@@ -78,7 +78,11 @@ Accounts can expose any interface in the future. This is the Miden version of a
 
 #### Customized note scripts
 
+<<<<<<< HEAD
+Users will be able to write their own note scripts using the Miden client. Note scripts are executed during note consumption and they can be arbitrarily complex due to the underlying Turing-complete Miden VM.
+=======
 Users can write their own note scripts using the Miden client. Note scripts are executed during note consumption and they can be arbitrarily complex due to the underlying Turing-complete Miden VM.
+>>>>>>> dd9667d79cf3d4cc5d6a7ccc8509bdf77fdc0289
```

Comment:
this was a conflict that got merged! Should be catching these before merging

Reply:
I don't understand why I didn't see that, did something get pushed in between me opening a branch off main and pushing? Possibly that's the reason ...

Reply:
This was a quick job too. 30 mins max.